workshop_app/views.py

```diff
@@ -195,102 +197,102 @@ def edit_profile(request):
-                               other_email=coordinator_email,
-                               phone_number=coordinator_phone,
-                               institute=coordinator_institute
-                               )
-
-                    # For Coordinator
-                    send_email(request, call_on='Booking Confirmed',
-                               workshop_date=workshop_date,
-                               workshop_title=workshop_title,
-                               other_email=coordinator_email,
-                               phone_number=instructor_phone
-                               )
-
-                elif action == 'change_date':
-                    cid = client_data.get('cid')
-                    new_workshop_date = datetime.strptime(client_data.get('new_date'), "%Y-%m-%d")
-                    coordinator_email = User.objects.get(id=cid)
-                    today = datetime.today()
-                    if today > new_workshop_date:
-                        # Invalid date
-                        pass
-                    else:
-                        workshop = Workshop.objects.filter(id=client_data.get('workshop_id'))
-                        workshop_date = workshop.first().date
-                        workshop.update(date=new_workshop_date)
-
-                        # For Instructor
-                        send_email(request, call_on='Change Date',
-                                   user_position='instructor',
-                                   workshop_date=str(workshop_date),
-                                   new_workshop_date=str(new_workshop_date.date())
-                                   )
 
-                        # For Coordinator
-                        send_email(request, call_on='Change Date',
-                                   new_workshop_date=str(new_workshop_date.date()),
-                                   workshop_date=str(workshop_date),
-                                   other_email=coordinator_email.email
-                                   )
+@login_required
+def workshop_status_instructor(request):
+    """ Workshops to accept and accepted by Instructor """
+    user = request.user
+    if not is_instructor(user):
+        return redirect(get_landing_page(user))
+    today = timezone.now().date()
+    proposed_workshop = Workshop.objects.filter(
+        instructor=user.id,
+        date__gte=today,
+    ).order_by('-date')
 
-            today = timezone.now()
-            proposed_workshop = Workshop.objects.filter(
-                instructor=user.id,
-                date__gte=today,
-            ).order_by('-date')
+    proposed_workshop_pending = Workshop.objects.filter(
+        status=0
+    ).order_by('-date')
 
-            proposed_workshop_pending = Workshop.objects.filter(
-                status=0
-            ).order_by('-date')
+    workshops = list(proposed_workshop) + list(proposed_workshop_pending)
```

Comment:
I reckon you are trying to merge two `querysets` here? Can you think of something better rather than converting each `queryset` to a list and then appending them together?